tests/app/questionnaire/test_value_source_resolver.py

```diff
@@ -1165,53 +1180,52 @@ def test_supplementary_data_value_source_list_items(
     )
 
 
-def test_supplementary_data_invalid_selector_raises_exception(
-    supplementary_data_store_with_data,
+def test_supplementary_data_value_source_list_items_value_missing_not_rendered(
```

Comment:
`not_rendered` wouldn't be correct here as the test does not check that. `Rendering` can only be checked by testing Placeholder Renderer or a functional here. Here, it would be better to use `value_missing_ignored`.